include/OsmAndCore/ITypefaceFinder.h

```diff
@@ -25,11 +30,28 @@ namespace OsmAnd
     public:
         struct OSMAND_CORE_API Typeface Q_DECL_FINAL
         {
-            Typeface(const sk_sp<SkTypeface>& skTypeface, hb_face_t* hbTypeface);
+            Typeface(const sk_sp<SkTypeface>& skTypeface,
+                     THbFontPtr hbFont_,
+                     std::set<uint32_t> delCodePoints,
+                     uint32_t repCodePoint);
+            Typeface(const sk_sp<SkTypeface>& skTypeface_,
+                     const THbFontPtr hbFont_);
+
+            static std::shared_ptr<Typeface> fromData(QByteArray data);
+
             ~Typeface();
 
+        public:
             sk_sp<SkTypeface> skTypeface;
-            std::shared_ptr<hb_face_t> hbTypeface;
+            THbFontPtr hbFont;
+
+            std::set<uint32_t> delCodePoints = {};//calculated deleting codepoint for current ttf
```

Comment:
Just because it's good programming manner to explicitly initialize variables whenever it possible. It's a good habit (my personal point at least).
Moreover  not initialized values like `int` will contain arbitrary value depends on compiler, compiler options, release/debug etc. thus it would be nice to initialize even it's not used right now. History knew a lot of stories when such not initialized values leads to serious bugs in future.
IMHO